test-runner/tests/connector-smoke-tests.yml

```diff
@@ -6,7 +6,49 @@
 -  config:
     -   testset: "Testing connection to iofog-connector via REST API"
 
-# FINISH ME
-#-  test:
-#    - name: "Check Connection"
-#    - group: "Controller"
\ No newline at end of file
+
+-  test:
```

Comment:
None of the tests that are expected to return 200 will actually pass (because of the issue identified above). The 404 tests will return "Success" because you are expecting them to fail.